src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -14,16 +14,13 @@
 final class EnumExtension extends AbstractExtension
 {
     /**
-     * @var TranslatorInterface
+     * @var Label
      */
-    private $translator;
+    private $label;
 
-    /**
-     * @param TranslatorInterface $translator
-     */
-    public function __construct(TranslatorInterface $translator = null)
+    public function __construct(Label $label)
```

Comment:
This is a BC-break. Let's fix this:
```suggestion
    public function __construct($label)
    {
        if ($label instanceof TranslatorInterface) {
            @trigger_error(sprintf(
                'Providing a %s instance to %s is deprecated and will not be supported in 5.0. Please provide a %s instance instead.'
                TranslatorInterface::class,
                __METHOD__,
                Label::class
            ), E_USER_DEPRECATED);
            $this->label = new Label($label);
            
            return;
        }
        $this->label = $label;
    }
```